environment.yml

```diff
@@ -0,0 +1,12 @@
+channels:
+- conda-forge
+# set nodefaults to remove the default channels
+# this is needed to keep memory use below RTD limit due to conda bug:
+# https://github.com/conda/conda/issues/5003
+- nodefaults
+dependencies:
+- pip
+- gdal==3.1.2
```

Comment:
I can imagine a time where we shift the gdal requirement for invest but forget to do it here and pip ends up trying to install a compatible gdal during `pip install natcap.invest`, and fails of course.

I'm not quite sure how we avoid that though...

Reply:
That's a good point. I can't think of a way around it though... you could `curl` https://github.com/natcap/invest/blob/main/requirements.txt and get the latest gdal requirement, but on RTD there's no way to do that before starting the build.

Reply:
I guess you could still do it in `conf.py`, but that's really supposed to be config for Sphinx, not general build setup.